src/esn.contact.libs/app/addressBook/addressbook-display.service.js

```diff
@@ -46,11 +46,7 @@ require('./addressbook.constants.js');
         var addressbookDisplayShell = new match.displayShell(addressbookShell);
 
         if (options.includeActions) {
-          var actions = _.filter(match.actions, function(action) {
-            return action.name === 'Settings' ? options.isSharingAddressbookEnabled : true;
-          });
```

Comment:
this action filter used to remove the settings menu button from the list when the address book sharing is disabled, which makes us lose (only accessible by URL, no button to go to the page ) the CalDAV URL field in the settings page ( and potentially the upcoming changes and added fields to the settings page ).

![image](https://user-images.githubusercontent.com/6764881/111462079-4917d780-871e-11eb-87a7-27b9d4f0d1d9.png)


this filter had no test case that is why none was included in this PR, the existing cases cover what needs to be tested.

Reply:
I think related test must be updated here

Reply:
the deleted changes had no tests before, the remaining code has test cases.